command/src/main/java/com/jagrosh/jdautilities/command/CommandBuilder.java

```diff
@@ -50,8 +50,8 @@
     private int cooldown = 0;
     private Permission[] userPermissions = new Permission[0];
     private Permission[] botPermissions = new Permission[0];
-    private final LinkedList<String> aliases = new LinkedList<>();
-    private final LinkedList<Command> children = new LinkedList<>();
+    private final ArrayList<String> aliases = new ArrayList<>();
+    private final ArrayList<Command> children = new ArrayList<>();
```

Comment:
Isn't it slightly better to use `List<Object> name = new ArrayList<>();` here?
Like:
```suggestion
    private final List<String> aliases = new ArrayList<>();
    private final List<Command> children = new ArrayList<>();
```